index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css\style.css">
+  </head>
+      <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script> -->
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Work</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Hey I am a mimic</h1>
```

Comment:
Please make sure to confirm that all linter errors are cleared before requesting a code review request. Having no linter errors is a requirement for all code reviews.

The linter errors will tell you what line of code is throwing the error so you can easily find the issue and correct it. I suggest running the linters from your command line periodically so you can see when an error is thrown before committing or pushing any changes. The process to run a linter check locally can be found here:

https://github.com/microverseinc/linters-config

- [ ]